st2api/tests/unit/controllers/v1/test_auth_api_keys.py

```diff
@@ -72,12 +72,47 @@ def test_get_all(self):
         self.assertEqual(len(resp.json), 5, '/v1/apikeys did not return all apikeys.')
 
         retrieved_ids = [apikey['id'] for apikey in resp.json]
-
         self.assertEqual(retrieved_ids,
                          [str(self.apikey1.id), str(self.apikey2.id), str(self.apikey3.id),
                           str(self.apikey4.id), str(self.apikey5.id)],
                          'Incorrect api keys retrieved.')
 
+    def test_get_all_with_pagnination_with_offset_and_limit(self):
+        resp = self.app.get('/v1/apikeys?offset=2&limit=1')
+        self.assertEqual(resp.status_int, 200)
```

Comment:
Please also add additional asserts to every test case for values of `X-Total-Count` and `X-Limit` header.

Besides that, LGTM :+1: